<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="modifieMenu">
    <property name="title">
     <string>Modifier</string>
    </property>
    <addaction name="deleteElementAction"/>
    <addaction name="addElementAction"/>
    <addaction name="setElementAction"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="connectionAction"/>
   </widget>
   <addaction name="modifieMenu"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="deleteElementAction">
   <property name="text">
    <string>Supprimer</string>
   </property>
  </action>
  <action name="connectionAction">
   <property name="text">
    <string>Connection</string>
   </property>
  </action>
  <action name="addElementAction">
   <property name="text">
    <string>Ajouter</string>
   </property>
  </action>
  <action name="setElementAction">
   <property name="text">
    <string>Modifier</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
